apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/derived/BasicDbTable.java

```diff
@@ -60,7 +63,7 @@ private int doMultiversionRollback(int height) {
         }
         long startTime = System.currentTimeMillis();
         String sql = "UPDATE " + table
-                + " SET latest = TRUE " + keyFactory.getPKClause() + " AND height ="
+                + " SET latest = TRUE " + (supportDelete() ? ", deleted = FALSE " : "") + keyFactory.getPKClause() + " AND height ="
```

Comment:
What about extract '(supportDelete() ? ", deleted = FALSE " : "")' into a method called like addtDeleteIfPresent to ease reading this query?